### PR TITLE
Clarify pod scheduling during node graceful termination

### PR DESCRIPTION
**Pull request by SergeyKanzhelev**
We never documented the Pod admission failures and Not Ready status. See https://github.com/kubernetes/enhancements/blob/80e76860ab0c9faa53fe833150091c15c5855ded/keps/sig-node/2000-graceful-node-shutdown/README.md?plain=1#L368-L371

/sig node
/assign @bobbypage 
/kind cleanup

### CODE REVIEW

**Review by sftim — COMMENTED** [review]
Thanks. I propose some tweaks - do they help?

File: content/en/docs/concepts/architecture/nodes.md [hunk 1]
```diff
@@ -396,11 +396,11 @@ The kubelet attempts to detect node system shutdown and terminates pods running
 
 Kubelet ensures that pods follow the normal
 [pod termination process](/docs/concepts/workloads/pods/pod-lifecycle/#pod-termination)
-during the node shutdown.
+during the node shutdown and new Pods are not being created.
 
 The Graceful node shutdown feature depends on systemd since it takes advantage of
 [systemd inhibitor locks](https://www.freedesktop.org/wiki/Software/systemd/inhibit/) to
-delay the node shutdown with a given duration.
+delay the node shutdown with a given duration. The feature is not working on Windows.
```
[the remark this answers is not in the document]
Comment (SergeyKanzhelev) [reply content/en/docs/concepts/architecture/nodes.md:404]: it was just clarification I noticed. Let me remove it

**Comment by SergeyKanzhelev** [comment]
@bobbypage can you please review when you have a chance to keep me honest here.

File: content/en/docs/concepts/architecture/nodes.md [hunk 2]
```diff
@@ -412,6 +413,20 @@ thus not activating the graceful node shutdown functionality.
...
+and does not schedule any Pods onto the affected node; other third-party schedulers are
+expected to follow the same logic. This means that new Pods won't be scheduled onto that node
+and therefore none will start.
+
+The kubelet **also** rejects Pods during the `PodAdmission` phase if an ongoing
+node shutdown has been detected, so that even Pods with a
+{{< glossary_tooltip text="toleration" term_id="toleration" >}} for
+`node.kubernetes.io/not-ready:NoSchedule` do not start there.
+
+At the same time when kubelet is setting that condition on its node via the API, the kubelet also begins
```
Comment (tengqm) [inline content/en/docs/concepts/architecture/nodes.md:427]:
```suggestion
At the same time when kubelet is setting that condition on its Node via the API, the kubelet also begins
```

File: content/en/docs/concepts/architecture/nodes.md [hunk 3]
```diff
@@ -412,6 +413,20 @@ thus not activating the graceful node shutdown functionality.
...
+Once systemd detects or notifies node shutdown, the kubelet sets a `NotReady` condition on
+the Node, with the `reason` set to `"node is shutting down"`. The kube-scheduler honors this condition
+and does not schedule any Pods onto the affected node; other third-party schedulers are
+expected to follow the same logic. This means that new Pods won't be scheduled onto that node
+and therefore none will start.
+
+The kubelet **also** rejects Pods during the `PodAdmission` phase if an ongoing
+node shutdown has been detected, so that even Pods with a
+{{< glossary_tooltip text="toleration" term_id="toleration" >}} for
+`node.kubernetes.io/not-ready:NoSchedule` do not start there.
```
Comment (tengqm) [inline content/en/docs/concepts/architecture/nodes.md:425]: This is good to know.
How about static Pods? If I or an automation tool happened to add a new manifest
or changing an existing one, without noticing that the node is being shut down,
what will happen?

Comment (SergeyKanzhelev) [reply content/en/docs/concepts/architecture/nodes.md:425]: the admissions will not allow anything: https://github.com/kubernetes/kubernetes/blob/53636bc7806d6e4fa1e9a4824d04779d0c4cde39/pkg/kubelet/nodeshutdown/nodeshutdown_manager_linux.go#L146-L157

**Comment by sftim** [comment]
/retitle Clarify pod scheduling during node graceful termination

**Pull request merged**